spec/lib/declare_schema/field_spec_spec.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+RSpec.describe 'DeclareSchema Model FieldSpec' do
+  before do
+    load File.expand_path('prepare_testapp.rb', __dir__)
+  end
+
+  context 'There are no model columns to change' do
+    it '#different_to should return false for int8 == int8' do
+      class Advert < ActiveRecord::Base
+        fields do
+          price :integer, limit: 8
+        end
+      end
+
+      subject = DeclareSchema::Model::FieldSpec.new(Advert, :price, :integer, { limit: 8, null: false })
```

Comment:
This is redundant with the class declaration above. At a minimum you could pull the field_spec from the class:
```ruby
subject = Advert.field_specs['price']
```
But it would be even cleaner if you could simply declare the `field_spec` here and skip the whole `ActiveRecord::Base` class (since the class is a higher-level concept; here we just want to test the `FieldSpec`).
Aha, it looks like you can, as long as you pass a `position:` option. And BTW, you don't need `{ }` around the options hash since it's the last argument.
```ruby
subject = DeclareSchema::Model::FieldSpec.new(Object, :price, :integer, limit: 8, null: false, position: 0)
```
With the above line, you can omit the `class Advert` entirely.

Same comment for the other tests below.